draft-ietf-taps-interface.md

```diff
@@ -214,8 +214,8 @@ implementing platform.
 We also make use of the following basic types:
 
 - Boolean: Instances take the value `true` or `false`.
-- Integer: Instances take positive or negative integer values.
-- Numeric: Instances take positive or negative real number values.
+- Integer: Instances take positive or negative integer values, or the value 0.
+- Numeric: Instances take positive or negative real number values, or the value 0.
```

Comment:
```suggestion
- Integer: Instances take integer values.
- Numeric: Instances take real number values.
```

Reply:
we probably shouldn't create a normative defintion of "integer" and "real" in a transport area document...